src/inject/dynamic-theme/image.ts

```diff
@@ -73,6 +73,9 @@ function analyzeImage(image: HTMLImageElement) {
         createCanvas();
     }
     const {naturalWidth, naturalHeight} = image;
+    if (naturalHeight > 0 && naturalWidth > 0) {
```

Comment:
Did you mean `naturalHeight === 0 || naturalWidth === 0`?

Also worth adding a warning `logWarn(`Image is empty ${url}`)` and checking if it is really empty.